### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by Tumetsu** [Crossbill-Highlights/crossbill-web#112]
Some easy UI accessibility improvements could be made such as:

- Improve contrast of the text
- Possibly add d&d alternative for moving tags between groups. For example separate edit mode, where clicking a tag opens a popup to select a destination group for it.
- Aria-label violations with span elements.

Tool to use for checking: Accessibility Insights for Web
